act-rules-format.bs

```diff
@@ -109,38 +109,34 @@ An aspect is a distinct part of the [test subject](#output-test-subject) or its
-
-The means by which the DOM tree is constructed, be it by a web browser or not, is not of importance as long as the construction follows the [Document Object Model](https://dom.spec.whatwg.org) [[DOM]].
-
-### CSS Styling ### {#input-aspects-css}
-
-The computed CSS [[CSS2]] styling resulting from parsing CSS and applying it to the DOM [[DOM]] may be of interest to ACT Rules that wish to test the web page as presented to the user. Through CSS styling, information about the position, the foreground and background colors, the visibility, and more, of elements becomes available.
-
-The means by which the CSS styling is computed, be it by a web browser or not, is not of importance as long as the computation follows any applicable specifications that might exist, such as the [CSS Object Model](https://www.w3.org/TR/cssom/) [[CSSOM]].
-
-### Accessibility Tree ### {#input-aspects-accessibility}
-
-The accessibility tree constructed from extracting information from both the DOM [[DOM]] tree and the CSS [[CSS2]] styling may be of interest to ACT Rules. This can be used to test the web page as presented to assistive technologies such as screen readers. Through the accessibility tree, information about the semantic roles, accessible names and descriptions, and more, of elements becomes available.
-
-The means by which the accessibility tree is constructed, be it by a web browser or not, is not of importance as long as the construction follows any applicable specifications that might exist, such as the [Core Accessibility API Mappings 1.1](https://www.w3.org/TR/core-aam-1.1/) [[CORE-AAM-1.1]].
-
-### Language ### {#input-aspects-text}
-
-Language, either written or spoken, contained in nodes of the DOM [[DOM]] or accessibility trees may be of interest to ACT Rules that intend to test things like complexity or intention of the language. For example, an ACT Rule might test that paragraphs of text within the DOM tree do not exceed a certain readability score or that the text alternative of an image provides a sufficient description.
+<div class=example>
+  <p>Test aspects for a rule that checks if images have an accessible name:</p>
+  <li>
+    <ul>DOM Tree</ul>
+    <ul>CSS Styling</ul>
+  </li>
+</div>
 
-The means by which the language is assessed, whether by a person or a machine, is not of importance as long as the assessment meets the criteria defined in [[wcag2-tech-req#humantestable]] [[WCAG]].
+<div class=example>
+  <p>Test aspects for a rule that checks if a transcript is available for videos:</p>
+    <li>
+    <ul>DOM Tree</ul
+    <ul>CSS Styling</ul
+    <ul>Audio output</ul
+    <ul>Visual output</ul
+  </li>
+</div>
 
+<div class=example>
+  <p>Test aspects for a rule that checks for use of (language specific) generic link texts like "click here" and "more":</p>
```

Comment:
Should this be switched out for an example where there is less controversy about how it should work or if a test for this can actually be created? 

Another possible example that would have the same test aspects: 

"Test aspects for a rule that checks if the language attribute for a web page matches the human language of the text on that page."

Reply:
I personally like the current example of checking for a transcript since it concedes that defining the test aspects may not always be black and white, and best judgement may be necessary. 

My only worry is that since it is an example in our document it may be seen as a "fully-complete" or "fully-correct" case that needs no further tweaking and will be referred to during implementation later on.